app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/DragAndDropSortInteractionViewModel.kt

```diff
@@ -135,8 +135,8 @@ class DragAndDropSortInteractionViewModel(
       dragDropInteractionContentViewModel.itemIndex = index
       dragDropInteractionContentViewModel.listSize = choiceItems.size
     }
-    // to update the content of grouped item
     (adapter as BindableAdapter<*>).setDataUnchecked(choiceItems)
+    adapter.notifyDataSetChanged()
```

Comment:
if we have diffUtil then why we need this?

With diffUtil we can get rid of `notifyDataSetChanged`

Reply:
Actually DIffUtil is unable to register merging changes and refresh the list, so to register those changes we need `notifyDataSetChanged`